src/components/select/select.ts

```diff
@@ -343,19 +342,19 @@ export class Select extends BaseInput<string[]> implements AfterViewInit, OnDest
 
       // If the user passed a cssClass for the select, add it
       selectCssClass += selectOptions.cssClass ? ' ' + selectOptions.cssClass : '';
-      overlay.setCssClass(selectCssClass);
+      this._overlay.setCssClass(selectCssClass);
 
-      overlay.addButton({
+      this._overlay.addButton({
         text: this.okText,
         handler: (selectedValues) => this.value = selectedValues
       });
 
     }
 
-    overlay.present(selectOptions);
+    this._overlay.present(selectOptions);
 
     this._fireFocus();
-    overlay.onDidDismiss((value: any) => {
+    this._overlay.onDidDismiss((value: any) => {
```

Comment:
instead of replacing overlay with `this._overlay' could you just do: `this._overlay = overlay` at the end of the function?
Also, since you are only calling `dismiss()` you could just declare `_overlay` as ViewController.